### PR TITLE
fixed indicator position

### PR DESCRIPTION
**Pull request by EgorSobko**
My Xcode thinks that I'm logged in from another account, that I created to upload test task for interview... I'll try to fix this.

### CODE REVIEW

File: PullToRefresh/PullToRefresh.swift [hunk 1]
```diff
@@ -199,20 +199,30 @@ class DefaultRefreshView: UIView {
...
+        frameInSuperview(superview)
         super.layoutSubviews()
     }
     
     override func willMoveToSuperview(newSuperview: UIView?) {
         super.willMoveToSuperview(newSuperview)
+        frameInSuperview(superview)
+    }
+    
+    private func frameInSuperview(newSuperview: UIView?) {
```
Comment (maksym-letiushov) [inline PullToRefresh/PullToRefresh.swift]: @EgorSobko could you plz rename this method to something like setupFrameInSuperview or similar ? It makes it more readable. Thanks.

**Comment by maksym-letiushov** [comment]
@EgorSobko also please update .podspec file. Set 'version' and 'tag' to 1.3. We will release a new pod version when this request is merged. Thanks.

File: PullToRefresh/PullToRefresh.swift (hunk 1 above)
Comment (EgorSobko) [inline PullToRefresh/PullToRefresh.swift]: Done.

**Comment by EgorSobko** [comment]
podspec updated

**Comment by maksym-letiushov** [comment]
thanks

**Pull request merged**